plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx

```diff
@@ -141,6 +142,8 @@ export const Stepper = (props: StepperProps) => {
     setFormState(current => ({ ...current, ...formData }));
   };
 
+  const currentStep = useTransformSchemaToProps(steps[activeStep], layouts);
```

Comment:
Think that we should make the second argument an object and have layouts be optional so that we can evolve the api over time.